README.md

```diff
@@ -62,7 +62,7 @@ is the front-end, and the back-end (or core) is now in
 **Build and Open:**
 
 ```bash
-> xcodebuild
+> xcodebuild -scheme "XiEditor"
```

Comment:
I'm not sure this is necessary; when I build with just `xcodebuild` it seems to still Just Work (CI and all?)

Reply:
I'm not totally sure how `xcodebuild` decides on the scheme, so I figured it was better to be more specific then have something unknown go wrong.

Reply:
Also: you should double check that this is true for you!

Reply:
I tracked down the man pages, which say:
> By default, xcodebuild builds the first target listed in the project

It seems like it would be safer to explicitly give a scheme rather than rely on no one to ever accidentally switching the order of the targets.

Reply:
I noticed that Travis also uses `-scheme "XiEditor"`, it would make sense for this to be consistent so if the order is ever changed, local builds wouldn't build the wrong target while Travis says that it's okay.